compiler/src/iree/compiler/Codegen/Dialect/UKernelOps.td

```diff
@@ -36,16 +36,17 @@ def IREECodegen_UKernelGenericOp :
     match directly into a function call with the arguments to the
     function call being the `ins`, `outs` and `other_operands`.
     
-    The operands of `memref` type are expected to lower to the following
-    argument sequence.
-    - `memref<f32> lowers to base_ptr.
-    - `memref<?xf32, <strides = [s0, 1], offsets = f>>
-          lowers to (base_ptr, f, index s0)
-    - `memref<?x?xf32, <strides = [s1, s0, 1], offsets = f>>
-          lowers to (base_ptr, f, s1, s0)
-    and so on....
+    The operands of `memref` type are expected to lower to this sequence of
+    function arguments:
+      `base_pointer, offset, stride0, stride1, stride2, ...`
+    where the number of strides passed as arguments is given by the optional
+    `strided_outer_dims` attribute if it is greater than or equal to the memref
```

Comment:
not sure I follow the "greater than or equal to memref rank" part.

Reply:
Clarified.